domain-sample-data/src/main/java/uk/gov/hmcts/cmc/domain/models/sampledata/SamplePayment.java

```diff
@@ -14,7 +14,9 @@ private SamplePayment() {
     public static PaymentBuilder builder() {
         return Payment.builder()
             .reference("RC-1524-6488-1670-7520")
-            .amount(new BigDecimal("4000"))
+            .amount(BigDecimal.valueOf(4000))
```

Comment:
same as above.. The concern is I can't see or make out easily what these changes relate to

Reply:
test coverage